src/Microsoft.SqlTools.ServiceLayer/Security/LoginData.cs

```diff
@@ -816,6 +914,7 @@ private class LoginPrototypeData : ICloneable
             private Microsoft.SqlServer.Management.Smo.Server server;
             private static string       defaultLanguageDisplay;
             private bool                windowsAuthSupported = true;
+            private bool                aADAuthSupported = false;
```

Comment:
nit: maybe either `aadAuthSupported` or `isAADAuthSupported` would read a little better.  Private value so no big deal.

Reply:
Yeah I hate it myself but didn't come up with a good alternative, will update later.